app/controllers/providers/no_national_insurance_numbers_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Providers
+  class NoNationalInsuranceNumbersController < ProviderBaseController
```

Comment:
This is just semantics, but I wonder if `MissingNationalInsuranceNumberWarningsController` might be more revealing?

It might be too verbose though, I'm not convinced either way.

Reply:
me neither 🤷 😄